core-services/image-mirroring/openshift/mapping_origin_4_10

```diff
@@ -133,6 +133,7 @@ registry.ci.openshift.org/origin/4.10:hyperconverged-cluster-operator quay.io/op
 registry.ci.openshift.org/origin/4.10:hyperconverged-cluster-operator-tests quay.io/openshift/origin-hyperconverged-cluster-operator-tests:4.10 quay.io/openshift/origin-hyperconverged-cluster-operator-tests:4.10.0 quay.io/openshift/origin-hyperconverged-cluster-operator-tests:latest
 registry.ci.openshift.org/origin/4.10:hyperconverged-cluster-webhook quay.io/openshift/origin-hyperconverged-cluster-webhook:4.10 quay.io/openshift/origin-hyperconverged-cluster-webhook:4.10.0 quay.io/openshift/origin-hyperconverged-cluster-webhook:latest
 registry.ci.openshift.org/origin/4.10:hyperkube quay.io/openshift/origin-hyperkube:4.10 quay.io/openshift/origin-hyperkube:4.10.0 quay.io/openshift/origin-hyperkube:latest
+registry.ci.openshift.org/origin/4.10:hypershift quay.io/openshift/origin-hypershift:4.10 quay.io/openshift/origin-hypershift:4.10.0 quay.io/openshift/origin-hypershift:latest
```

Comment:
Why mirror to `quay.io/openshift/origin-hypershift`, shouldn't that be `registry.ci.openshift.org/hypershift/hypershift:latest`?

Reply:
I'm not sure, that change was automatically generated by `make openshift-image-mirror-mappings` after I added the hypershift file...

Reply:
Everything that is part of origin/ocp is part of the same namespace and published the same way.  You can publish to multiple locations.  We don't allow images in OCP to have a canonical source that doesn't follow the pattern (it's a bit like RPMs - you can put an RPM in multiple repos, but the canonical location is decided by the publisher of the RPM).